proxy/envoy/fault.go

```diff
@@ -40,14 +40,19 @@ func buildFaultFilters(routeConfig *HTTPRouteConfig) []HTTPFilter {
 
 // buildFaultFilter builds a single fault filter for envoy cluster
 func buildHTTPFaultFilter(cluster string, faultRule *proxyconfig.HTTPFaultInjection, headers Headers) *HTTPFilter {
+	abort, delay := buildAbortConfig(faultRule.Abort), buildDelayConfig(faultRule.Delay)
```

Comment:
Also the following reads better to me :)
abort := buildAbortConfig(faultRule.Abort)
delay := buildDelayConfig(faultRule.Delay)